packages/core/src/ceramic.ts

```diff
@@ -247,13 +264,14 @@ export class Ceramic implements CeramicApi {
       this._logger,
       this.dispatcher,
       modules.anchorService.validator,
-      this._streamHandlers,
-      this.context
+      this,
```

Comment:
Full `this`? Nothing more concrete?

Reply:
Not in this PR. Was getting pretty big.

I can do another PR where I can pull out functionality so we're a little less circular.